freeipa/src/main/java/com/sequenceiq/freeipa/client/FreeIpaClient.java

```diff
@@ -70,49 +72,18 @@ public void userSetPassword(String user, String password) throws FreeIpaClientEx
         userMod(user, "setattr", "krbPasswordExpiration=20380101000000Z");
```

Comment:
I am the perpetrator of the hack to set the password expiration to a time in 2032. Maybe it's time to revisit this; we're on a newer version of the IPA, and it might accept an expiration value of 0 to mean "never expire."

Reply:
added to DISTX-144